modules/ROOT/pages/_partials/sync/provisioning.adoc

```diff
@@ -1,37 +1,62 @@
 
+It's neccessary to wrap your Sync Server application into Docker container and publish your container to a Docker registry. 
+The easiest way is to use a public one such as link:https://hub.docker.com/[Dockerhub].
 
-:service-name: Data Sync
+= Preparing for deployment
 
-= Provisioning {service-name} Service
+{sync-service} application is a Node.js project, to wrap it into a container, create `Dockerfile` in the project's directory like this:
 
-[discrete]
-== Prerequisites
-include::../generic-provisioning-pre-req.adoc[]
+[source,dockerfile]
+----
+FROM node:8
+WORKDIR /usr/src/app
+# copy Node.js specific files required for Node.js Data Sync application
+COPY package*.json ./
+# copy Data Sync Server app source file to the workdir (add all other files that are needed) 
+COPY server.js . 
+RUN npm install
+# TCP port that Sync Server app is listening on
+EXPOSE 4000
+CMD [ "npm", "start" ]
+---- 
+
+Build the Docker container:
+[source,bash]
+----
+$ docker build .
+----
+
+Push to the Dockerhub repository :
+[source,bash]
+----
+$ docker push <your-repo>/<container-name>
```

Comment:
@wojta  I say let's not do this at all. How to build a container is beyond the scope of this doc. It's enough to say "you first need to build a container for your app and push it to a container registry. For more details about how to do it on Openshift, please check the doc (link to s2i doc perhaps).

We may add support for s2i in the future, which will make this section a bit redundant. So let's not worry about this too much.